src/main/java/com/quantumsoft/qupathcloud/entities/Dataset.java

```diff
@@ -18,62 +18,108 @@
 import com.fasterxml.jackson.annotation.JsonIgnore;
 import java.util.Objects;
 
+/**
+ * The type Dataset.
```

Comment:
This does not really add any additional information - specify that the Dataset represents or remove the comment...